alternatives.8

```diff
@@ -266,6 +266,10 @@ Tell which version of
 .B alternatives
 this is (and give some usage information).
 .TP
+.B --keep-missing
+Allow user to keep files which aren't available in newer version of program (e.g. java).
```

Comment:
I would say something like:
When switching between alternatives, if the new variant does not provide some files, keep the links pointed to the previous implementation. 
And I would remove the java reference.

Reply:
Thank you.